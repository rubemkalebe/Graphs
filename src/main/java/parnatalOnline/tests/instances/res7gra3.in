7 3
26 17 21 6 24 12
19 10 17 12 27
5 4 29 5
15 21 28
25 19
19
