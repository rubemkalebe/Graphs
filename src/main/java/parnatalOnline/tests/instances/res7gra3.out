42.000000
0 4, 1 3, 1 5, 2 3, 2 4, 2 6, 
14070
0.012086;0.000000;